### PR TITLE
Fixed wiki build documents

### LINKED ISSUE #1449: Seed database is out of date
The current seed database does not work with the current code.

We need to download the current seed database:  http://cbio.mskcc.org/cancergenomics/public-portal/downloads/cbioportal-seed.sql.gz

apply the migration script: https://github.com/cBioPortal/cbioportal/blob/master/docs/Updating-your-cBioPortal-installation.md

And, then replace the seed database.

### PR DESCRIPTION
**Pull request by ecerami**
# What? Why?

Documentation changes to fix the build process.

Fixes #1449, #1447, #1446

Changes proposed in this pull request:

documentation changes

# Notify reviewers
Notifying @jjgao

### CODE REVIEW

File: docs/Downloads.md [hunk 1]
```diff
@@ -11,16 +10,10 @@ This page describes the various files available for download.
 
 This file is a MySQL database dump for seeding a new instance of the cBioPortal.  It contains all the necessary background data for a properly functioning cBioPortal website, including cancer types, gene, uniprot-mappings, drug and network data.  The file is available for download here:
 
-[cbioportal-seed.sql.gz](http://cbio.mskcc.org/cancergenomics/public-portal/downloads/cbioportal-seed.sql.gz)
+[cbioportal-seed.sql.gz](https://storage.googleapis.com/cbiostorage-1/cbioportal-seed.sql.gz)
```
Comment (pieterlukasse) [inline docs/Downloads.md]: :+1: 
One request, maybe for later: I would like to see this versioned. So maybe at some point we could have a list of links here like 

```
[cbioportal-seed.sql.gz for v1.2.0](https://storage.googleapis.com/cbiostorage-1/seed/1.2.0/cbioportal-seed.sql.gz)
[cbioportal-seed.sql.gz for v1.2.2](https://storage.googleapis.com/cbiostorage-1/seed/1.2.2/cbioportal-seed.sql.gz)
[cbioportal-seed.sql.gz for v1.2.2](https://storage.googleapis.com/cbiostorage-1/seed/1.3.0/cbioportal-seed.sql.gz)
etc
```

File: docs/Deploying.md [hunk 2]
```diff
@@ -44,25 +60,25 @@ After doing this, you can look in the tomcat log file (`$CATALINA_HOME/logs/cata
...
+## Verify the Web Application
 
 Lastly, open a browser and go to:  
 
 [http://localhost:8080/cbioportal/](http://localhost:8080/cbioportal/)
 
-# Important
+## Important
 
 Each time you add new data or modify any code, you must redeploy the WAR file.
```
Comment (pieterlukasse) [inline docs/Deploying.md]: maybe change to 

- "Each time you modify any code, you must recompile and redeploy the WAR file."
- "Each time you modify any properties (see customization options), you must recompile and redeploy the WAR file."
- "Each time you add new data, you must restart tomcat."

File: docs/Pre-Build-Steps.md [hunk 3]
```diff
@@ -1,28 +1,32 @@
...
+## Prepare Property Files
 
 The portal requires two properties files:  one for global configuration (`portal.properties`) and one for logging (`log4j.properties`).  Example files are available within GitHub, but you must take the following steps to prepare them.
 
     cd src/main/resources
     cp portal.properties.EXAMPLE portal.properties
     cp log4j.properties.EXAMPLE log4j.properties
 
-For more information about the portal.properties file, see the following [reference](portal.properties-Reference.md) page.
+For more information about the `portal.properties` file, see the following [reference](portal.properties-Reference.md) page.
```
Comment (pieterlukasse) [inline docs/Pre-Build-Steps.md]: One thing we are missing here is are the other configuration files that are available in cBioPortal. Would be nice to at least mention them here and link to a page that describes them. The files are:
- /cbioportal/core/src/main/resources/cbio_cancer_genes.txt
- /cbioportal/core/src/main/resources/gene_sets.txt (?)
- and perhaps some other files in /cbioportal/core/src/main/resources/ (?)

And perhaps later they should be added to seed DB and follow its update cycle?

**Comment by ecerami** [comment]
@pieterlukasse:  thanks for the feedback.  Can you consider adding some of these as separate tickets?  (as I won't have time this week to specifically address, and I would like to get this pushed out ASAP so that the build process is working again).  thanks.

File: docs/Pre-Build-Steps.md (hunk 3 above)
Comment (pieterlukasse) [inline docs/Pre-Build-Steps.md]: @ecerami @awernibi: logged this in a separate issue: #1484

File: docs/Downloads.md (hunk 1 above)
Comment (pieterlukasse) [inline docs/Downloads.md]: This request was also made at #767

File: docs/Deploying.md (hunk 2 above)
Comment (pieterlukasse) [inline docs/Deploying.md]: @ecerami : I will ask @oplantalech to make this enhancement.

File: docs/Load-Sample-Cancer-Study.md [hunk 4]
```diff
@@ -1,50 +1,52 @@
...
+cd <your_cbioportal_dir>/core/src/main/scripts/importer
+```
 
-Lastly, load the case sets and clinical attributes:
+and then run the following command:
 
-    $PORTAL_HOME/core/src/main/scripts/cbioportalImporter.py --jvm-args "-Dspring.profiles.active=dbcp -cp $CONNECTOR_JAR:$CORE_JAR" --command import-case-list --meta-filename portal-study/case_lists
-    $PORTAL_HOME/core/src/main/scripts/cbioportalImporter.py --jvm-args "-Dspring.profiles.active=dbcp -cp $CONNECTOR_JAR:$CORE_JAR" --command import-study-data --meta-filename portal-study/meta_clinical.txt --data-filename portal-study/data_clinical.txt
+```
+./metaImport.py -s ../../../test/scripts/test_data/study_es_0/ -n -o
```
Comment (pieterlukasse) [inline docs/Load-Sample-Cancer-Study.md]: for this to run, you (today) will still need an environment variable called PORTAL_HOME pointing to the root of the source checkout.

Comment (pieterlukasse) [inline docs/Load-Sample-Cancer-Study.md]: never mind. I see you have this in another page.

**Comment by pieterlukasse** [comment]
@ecerami no problem. I made or linked tickets (or people) to each of the topics. So you can go ahead with this PR and merge it.

File: docs/Deploying.md (hunk 2 above)
Comment (oplantalech) [inline docs/Deploying.md]: I like Pieter option since it is much clear to understand. However, I would rephrase the first point:

- "Each time you modify any java code, you must recompile and redeploy the WAR file."

**Comment by jjgao** [comment]
@ecerami was this one replaced with #1475 to hotfix? Should we close this without merge to rc?

**Comment by ecerami** [comment]
@jjgao yes, I am closing this one.
